src/server.ts

```diff
@@ -380,12 +380,17 @@ const createParticipant = async (tag: string): Promise<ParticipantInfo> => {
 const deleteSession = async () => {
   if (sessionId) {
     try {
-      await webRTCController.deleteSession(accountId, sessionId);
       console.log(`Deleted WebRTC session: ${sessionId} `);
       sessionId = "";
     } catch (e) {
+      let error: any = e;
```

Comment:
you should be able to type cast this as `Error` so 

`let error: Error = e;`

Reply:
good point - thanks.  I'll evolve it

Reply:
question on the package-lock.json comment.  package-lock.json changed because I suspect there are dependency version updates.  package.json and package-lock.json were not included in .gitignore.  Bottom line - I don't understand the issue.

Reply:
Oh there were so many deletions I assumed it was deleted. Looks like it wasn't entirely deleted, but I suppose some things updated. The package-lock does get filled with the exact dependencies, and the dependencies of those dependencies with their corresponding versions "locked in" hard.

Reply:
it looks fine to me! Just was curious why it deleted almost 20k lines

Reply:
I cleaned up the typing of the catch clause (e) parameter, and that also caused me to find and fix errors in the console.log outputs in one of the catch clauses, and to find and fix a missing call to actually delete the session.